pkg/querier/blocks_scanner_test.go

```diff
@@ -162,7 +162,7 @@ func TestBlocksScanner_StopWhileRunningTheInitialScanOnManyTenants(t *testing.T)
 	time.Sleep(time.Second)
 
 	stopTime := time.Now()
-	require.NoError(t, services.StopAndAwaitTerminated(context.Background(), s))
+	_ = services.StopAndAwaitTerminated(context.Background(), s)
```

Comment:
I've noticed this and the following test are flaky because in this case `StopAndAwaitTerminated()` may either return `nil` or `context.Canceled`.